backend/app/api/endpoints/openapi_responses.py

```
@@ -145,7 +148,591 @@ def _task_to_response_object(
+            messages = MessageConverter.build_messages(
+                history=history,
+                current_message=input_text,
+                system_prompt=system_prompt,
+            )
+
+            # Create LangChain model from config
+            try:
+                llm = LangChainModelFactory.create_from_config(
+                    model_config, streaming=True
+                )
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)